presets/workspace/models/metadata.go

```diff
@@ -0,0 +1,38 @@
+package models
+
+import (
+	_ "embed"
+
+	"gopkg.in/yaml.v2"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/kaito-project/kaito/pkg/model"
+)
+
+var (
+	//go:embed supported_models.yaml
+	supportedModelsYAML []byte
+
+	// SupportedModels is a global map that holds the source of truth
+	// for all supported models and their metadata.
+	SupportedModels map[string]*model.Metadata
```

Comment:
You would need a sync.Mutex to protect the map. You can add it later though.

Reply:
Depends on whether go test -race would complain or not. Leave it for future change to validate.